middlewares/requireLogin.js

```diff
@@ -2,19 +2,25 @@ const { verify } = require('../helpers/token');
 const errorCodes = require('../helpers/errorCodes');
 
 module.exports = async (req, res, next) => {
+    let claims;
 
-	let claims;
-	try {
-		claims = verify(req.headers.token);
-	} catch {
-		return res.status(401).send({ error: errorCodes['E0001'] });
-	}
+    try {
+        const authHeader = req.headers.authorization;
```

Comment:
Wont changing this break all frontend calls that currently use the "token" header instead of the "authorization" header?

Using the authorization header with bearer tokens is definitely the best approach, but we should maybe continue allowing the use of the "token" header for now.